Examples/Docs/Sources/SyncExamples/main.swift

```diff
@@ -165,3 +165,42 @@ private func transactions() throws {
     }
     // End Transactions Retry Example 3
 }
+
+private func versionedAPI() throws {
+    let uri = "mongodb://localhost:27017"
+
+    do {
+        // Start Versioned API Example 1
+
+        // Declare API version "1" for the client
+        var clientOpts = MongoClientOptions()
+        clientOpts.serverAPI = MongoServerAPI(version: .v1)
+        let client = try MongoClient(uri, options: clientOpts)
+        // End Versioned API Example 1
+    }
+
+    do {
+        // Start Versioned API Example 2
+
+        // Use the `strict` option
+        var opts = MongoClientOptions()
+        opts.serverAPI = MongoServerAPI(version: .v1, strict: true)
+        let client = try MongoClient(uri, options: opts)
+
+        var findOpts = FindOptions()
+        findOpts.cursorType = .tailable
+        // Fails with an error because `tailable` is not part of version 1
+        let cursor = try client.db("db").collection("coll").find(options: findOpts)
```

Comment:
`try`/`catch` here to match the previous example?

Reply:
updated (ironically, I removed the try catch from the other example because of your comment there)